terraform/20-app/s3.ingest.tf

```diff
@@ -67,7 +67,21 @@ module "s3_ingest" {
           "${module.s3_ingest.s3_bucket_arn}/processed/*.json",
         ]
       }
-    ]
+    ],
+        local.is_ready_for_etl ? [
+        {
+          Sid    = "AllowCrossAccountAccessFromETLPublisherLambda",
+          Effect = "Allow",
+          Principal = {
+            AWS = "arn:aws:iam::${var.etl_account_id}:role/${local.project}-etl-${local.environment}-publisher"
```

Comment:
This allows the lambda with the name `uhd-etl-<env vame>-publisher` in the corresponding sibling AWS etl account to push files into this ingest bucket